event_routing_backends/tests/test_helpers.py

```diff
@@ -83,3 +88,24 @@ def test_get_course_from_id_unknown_course(self, mock_get_course_overviews):
         mock_get_course_overviews.return_value = []
         with self.assertRaises(ValueError):
             get_course_from_id("foo")
+
+    @data("edx", "10228945687")
+    def test_get_user_by_username(self, username):
+        """Test that the method get_user returns the right user based on given username parameter.
+
+        Expected behavior:
+            - Returned user corresponds to the username.
+        """
+        user = get_user(username)
+
+        self.assertEqual(username, user.username)
+
+    def test_get_user_by_id(self):
+        """ Test that the method get_user returns the right user based on the user id.
+
+        Expected behavior:
+            - Returned user is the edx_user
+        """
+        user = get_user(self.edx_user.id)
+
+        self.assertEqual(self.edx_user, user)
```

Comment:
Add a new test with two users, the first user with a hardcoded ID and the second one with the same ID as the username, verify that getting the user by the user_id as str returns the second user

Reply:
Done